query-node/mappings/src/workingGroups.ts

```diff
@@ -725,6 +727,16 @@ export async function workingGroups_LeadRemarked({ store, event }: EventContext
       return invalidMetadata(`Forum post not found by id: ${postId}`)
     }
     await moderatePost(store, event, 'leadRemark', post, actor, rationale)
+  } else if (metadata?.verifyValidator) {
+    if (group.name !== 'membershipWorkingGroup') {
+      return invalidMetadata(`The ${group.name} can't verify the validator's membership`)
+    }
+    const { memberId, isVerified } = metadata.verifyValidator
+
+    const member = await getMembershipById(store, createType('u64', Number(memberId)))
```

Comment:
```suggestion
    const member = await getById(store, Membership, memberId)
    // If member does not exist, just log an invalid metadata message
```

Since there is no runtime validation for metaprotocol messages, as a general rule, the functions that throw should not be used in metaprotocol mappings logic. 

These functions should only be called in the mappings when we know for sure that the corresponding state exists in the runtime so if the QN db does not have that state, it means there is a state consistent and hence throw the error.

cc. @thesan